android/src/main/java/com/reactnative/googlecast/api/RNGCSessionManager.java

```diff
@@ -20,16 +20,15 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class RNGCSessionManager
-  extends ReactContextBaseJavaModule implements LifecycleEventListener, SessionManagerListener<CastSession> {
+public class RNGCSessionManager extends ReactContextBaseJavaModule
```

Comment:
Is there a lint command strategy for Java so I can revert these?

Reply:
I actually wonder that myself. There isn't any set right now but we could certainly use one.